### ISSUE TITLE
Right pinky finger

### ISSUE THREAD

**Issue by hasufell** [workman-layout/Workman#57]
The web page says the load on the right pinky finger is reduced compared to colemak, but I don't see why. 'I' is slightly more frequent than 'O'.

**Comment by yg8ijvjvjv** [comment]
You might be using a flawed corpus, usually too small and unvaried, commonly just a couple of books. Have you tried checking the frequency with a modern balanced corpus?

**Comment by hasufell** [comment]
I'm not using any corpus. I believe smashing together a set of literature is in no way representative to the everyday use of a keyboard, such as writing text on github issues, social media and chatting.

Those are mostly *simple english*. Also see my comment here: https://github.com/workman-layout/Workman/issues/59#issuecomment-688157555

**Comment by yg8ijvjvjv** [comment]
Workman is not specifically designed to your specific everyday use of a keyboard.

**Comment by hasufell** [comment]
I don't think that was my argument. My argument was that simple english has a different heat map.

If you use 500.000 words of shakespear for your analysis, i don't think the result and the conclusions will be very useful to most users.

**Comment by JasonCarvalho-tech** [comment]
Maybe specifically showing what data set you are using to come to this conclusion would be helpful.
